src/Command/ErrorFormatter/TableErrorFormatter.php

```diff
@@ -71,6 +74,9 @@ public function formatErrors(
 					$tip = str_replace('%configurationFile%', $projectConfigFile, $tip);
 					$message .= "\n💡 " . $tip;
 				}
+				if (is_string($this->editorUrl)) {
+					$message .= "\n    " . str_replace(['%file%', '%line%'], [$file, (string) $error->getLine()], $this->editorUrl);
```

Comment:
What about https://github.com/phpstan/phpstan/issues/4984#issuecomment-835847262?

Reply:
Sorry, I can't see what should be escaped here :-( Ping @Kocal

Reply:
I mean using the same escaping codes than here https://github.com/symfony/symfony/blob/db750ed8ae80de9417e7964c532ab4bc44b4f9d4/src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php#L203 (`\033]8;;`) to nicely print URLs in the console

Reply:
What does it do exactly? Add some color? It's not working for me => Why don't you just make a PR yourself and add it?!

Reply:
Using the escaping codes `\033]8;;` or `\e]8;;` allow you to nicely prints URL in the console, see https://github.com/ostrolucky/symfony/blob/db750ed8ae80de9417e7964c532ab4bc44b4f9d4/src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php#L203 and https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda.

This is what happens in my terminal:
![image](https://user-images.githubusercontent.com/2103975/118398062-47ec2380-b657-11eb-9833-5d64dca1ad99.png)

By using them, you can display a link `Open in IDE` instead of `atom://core/open/file?filename=/.../src/Controller/FooController.php&line=69` which would be a lot user friendly.

WDYT?

Reply:
My terminal ("Konsole") doesn't support that. And since the list at https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#feature-requests-sent is quite long, it looks like this isn't widely adopted yet. (Besides, I'm not convinced that "hiding" URLs from users that way is generally a good idea at all ;-)

But the "strongest" argument against it is: If somebody wants that, they can easily set it up themselves: The config is
```neon
editorUrl: 'atom://core/open/file?filename=%%file%%&line=%%line%%'
```
So in principle it should work if you change that to something like:
```neon
editorUrl: '\033]8;;atom://core/open/file?filename=%%file%%&line=%%line%%\033\\Open in IDE\033]8;;\033\\'
```
So in total I'd say: Let's merge this now as-is. Then I'll write the documentation somewhere at https://phpstan.org/config-reference, and there we can explain this too.